nifs/dockerfiles/arcade-nifty/arcade_activate

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+
+if [ ! -f "/scratch/login.cl" ]; then
+	ln -s /iraf/login.cl /scratch/login.cl
+fi
+
+source activate iraf27
+
+# To make container useful in both interactive sessions
+# and batch processing, check if xterm is available.
+# If so, start up xterm. If not, get ready for
+# incoming processing command.
+if xhost >& /dev/null; then 
+	# Display exists
+	xterm -T $1
+else 
+	# Display invalid"
+	exec "$@"
+fi
+
```

Comment:
I _think_ this will work by only testing will tell.. sorry, this kind of thing will be easier in skaha.